packages/style-kit/src/_inputs.scss

```diff
@@ -20,46 +32,90 @@
   margin-bottom: -$grid;
 }
 
-@mixin search-input-container($theme) {
-  padding: 0px $grid;
-  border-radius: $border-radius-large;
-  border: 1px solid map-get($theme, 'light');
-  background-color: map-get($theme, 'bgShade');
-  display: flex;
-  align-items: center;
-  max-width: 360px;
+@mixin attachment($theme) {
+  @include input($theme);
+  background-color: transparent;
+  border-width: 1.5px;
+  display: inline-flex;
+  flex-direction: column;
+  width: 60%;
+  padding: $grid / 2 $grid * 8 $grid / 2 $grid / 2;
+  .content {
+    display: flex;
+    align-items: center;
+    @include item-spacing();
+    .icon {
+      flex-shrink: 0;
+      align-self: flex-start;
+    }
+  }
+}
+
+@mixin date-value($theme) {
+  @include input($theme);
+  width: 60%;
+  padding-right: $grid * 8;
+  background-image: url($calendar-svg);
+  background-repeat: no-repeat, repeat;
+  background-position: right 0.7em top 50%, 0 0;
 }
 
 @mixin search-input {
-  @include body-text-medium;
-  width: 100%;
-  border: 0;
-  outline: none;
-  background-color: transparent;
+  border-radius: $border-radius-large;
+  min-width: 60%;
+  max-width: fit-content;
+  background-image: url($search-svg);
+  background-repeat: no-repeat, repeat;
+  background-position: left 0.7em top 50%, 0 0;
+  text-indent: $grid * 4;
+}
+
+@mixin dropdown-input($theme) {
+  @include text-input($theme);
+  min-width: 210px;
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  appearance: none;
+  background-image: url($down-arrow-svg);
+  background-repeat: no-repeat, repeat;
+  background-position: right 0.7em top 50%, 0 0;
 }
 
 @mixin inputs-main($theme) {
   input {
-    margin: $grid * 2;
     display: block;
     @include text-input($theme);
     &:focus {
-      outline-color: map-get($theme, 'light');
+      outline-color: map-get($theme, 'highlight');
     }
-    &:placeholder {
+    &::placeholder {
+      @include body-text-medium;
       color: map-get($theme, 'placeholder');
     }
+    &[type='search'] {
+      @include search-input;
+      &:focus {
+        background-size: 5%;
+        outline: none;
+        border: 1.5px solid map-get($theme, 'highlight');
+      }
+    }
   }
   label {
     @include input-label;
     display: block;
   }
-  .search-bar {
+  .dropdown-input {
     margin: $grid * 2;
-    @include search-input-container($theme);
-    input[type='search'] {
-      @include search-input;
-      margin: $grid;
+    @include dropdown-input($theme);
+    &:focus {
+      outline-color: map-get($theme, 'highlight');
     }
   }
+  .attachment {
+    @include attachment($theme);
+  }
+  .date-value {
```

Comment:
I think keep this in the `input` scope as 

```
&[type='date']
```

Reply:
except in this case this "date-value" is being used to display the field elements in view mode - so it's not an input it just has the same style properties as input so I included them in its mixin.

Reply:
It's still input-ish though?

![image](https://user-images.githubusercontent.com/9100419/87277947-850cf400-c522-11ea-8357-b0e18996a842.png)

I would think if you wanted it to be read-only you could just disable the input:

```
<input type="date" value="2020-07-13" disabled="true">
```